python-package/securexgboost/core.py

```diff
@@ -2715,6 +2408,202 @@ def get_split_value_histogram(self, feature, fmap='', bins=None, as_pandas=True)
                 "Returning histogram as ndarray (as_pandas == True, but pandas is not installed).")
         return nph
 
+##########################################
+# Enclave init and attestation APIs
+##########################################
+
+def init(enclave_image=None, remote_addr=None, user_name=None, 
+        sym_key_file=None, priv_key_file=None, cert_file=None,
+        log_verbosity=0):
+    """
+    Parameters
+    ----------
+    enclave_image: str
+        Path to enclave binary
+    user_name : string
```

Comment:
For the parameter type (`string`), we should probably try to be consistent, i.e. choose `str` or `string`. I think most other functions use `str`, so let's change `string` to `str` for these parameters?